lib/util.js

```diff
@@ -369,6 +369,32 @@ exports.getVideoDescription = (html) => {
     new Entities().decode(exports.stripHTML(description[1])) : '';
 };
 
+/**
+ * Get video extras from html
+ * 
+ * @param {String} body
+ * @return {Object}
+ */
+const extrasRegexp = /<h4 class="title">[\s\S]*?([\s\S]*?)<\/h4>[\s\S]*?<ul.*?class=".*?watch-info-tag-list\">[\s\S]*?<li>([\s\S]*?)<\/li>/g;
```

Comment:
Using `.exec()` below changes the state of this regexp. Define it inside `getVideoExtras()` so that a fresh state is used every time. Otherwise, it will only find extras for the first video.

Reply:
Regexp doesn't need the first `[\s\S]*?`, `"` doesn't need to be escaped